biome.json

```diff
@@ -100,5 +101,17 @@
     "formatter": {
       "indentWidth": 2
     }
-  }
+  },
+  "overrides": [
+    {
+      "include": ["src/**"],
+      "linter": {
+        "rules": {
+          "correctness": {
+            "useHookAtTopLevel": "off"
+          }
```

Comment:
I think it might be better to switch this one, so that the general rule is off, but it's error in the front end directory. Primarily because this will also impact "/website" (the docs). But maybe that's good? 🤷🏼

Reply:
This way it works when you run in `frontend` as root. I didn't get any errors in `website`, and it's easy to fix when we do